README.md

```diff
@@ -0,0 +1,34 @@
+# ECosmetics REST API
+
+###  Set up the project as below
+     1. Checkout the project from Git Repo
+     2. Import the project into IDE 
+     3. Install Virtual environment
+        py -m pip install virtualenv
+     4. Create virtual environment
+        python -m venv c:\path\to\myenv
```

Comment:
Minor:

This could be made even more generic, like: 

`python -m venv <path_to_my_env>`